website/content/v1.3/reference/cli.md

```diff
@@ -96,7 +96,7 @@ talosctl cluster create [flags]
       --bad-rtc                                  launch VM with bad RTC state (QEMU only)
       --cidr string                              CIDR of the cluster network (IPv4, ULA network for IPv6 is derived in automated way) (default "10.5.0.0/24")
       --cni-bin-path strings                     search path for CNI binaries (VM only) (default [/home/user/.talos/cni/bin])
-      --cni-bundle-url string                    URL to download CNI bundle from (VM only) (default "https://github.com/siderolabs/talos/releases/download/v1.3.0-beta.1/talosctl-cni-bundle-${ARCH}.tar.gz")
+      --cni-bundle-url string                    URL to download CNI bundle from (VM only) (default "https://github.com/siderolabs/talos/releases/download/v1.3.0-beta.2-dirty/talosctl-cni-bundle-${ARCH}.tar.gz")
```

Comment:
-dirty

Reply:
no way :facepalm:

Reply:
thank you, I always miss whether it's `make generate docs` or `make docs generate`. Should be the latter.